packages/crayons-extended/platform-ui/src/utils/utils.ts

```diff
@@ -0,0 +1,25 @@
+export function format(first: string, middle: string, last: string): string {
```

Comment:
please remove the `format` function

Reply:
This function is used in the _plop-template_ hbs files. These are available in crayons-core too.